server/api/src/db/requests/views.py

```diff
@@ -7,7 +7,7 @@ def create():
         CREATE MATERIALIZED VIEW map AS (
             SELECT
                 srnumber,
-                createddate,
+                createddate::date,
                 requesttype,
```

Comment:
So this code will only run when someone initializes the DB, not when it's updated. Which means we'll have to update the `dev` and `prod` servers manually to change the type of the column. (Another reason to get migrations working!) Let's discuss the benefits of this change.

Reply:
ah! didn't mean to check this in. it was part of some performance testing i was doing. i'll back this out. good catch!

we may want to do something like this in the future but now is probably not the time

Reply:
ok